tests/test_sensor.py

```diff
@@ -80,6 +82,49 @@ def status_func(value):
     assert sensor.status == Sensor.Status.NOMINAL
 
 
+def test_sensor_value_setter(diverse_sensors):
```

Comment:
```suggestion
def test_sensor_value_setter(diverse_sensors) -> None:
```
Otherwise mypy doesn't check the function.